src/applications/check-in/components/pages/ConfirmablePage/index.jsx

```diff
@@ -15,13 +26,35 @@ const ConfirmablePage = ({
   isEditEnabled = false,
   LoadingMessage = () => <va-loading-indicator message="Loading..." />,
   Footer,
+  currentlyLoggedIn,
 }) => {
+  const selectCurrentContext = useMemo(makeSelectCurrentContext, []);
+  const { token } = useSelector(selectCurrentContext);
   useEffect(() => {
     focusElement('h1');
   }, []);
-  const editHandler = useCallback(dataToEdit => {
-    dataToEdit.editAction(dataToEdit);
-  }, []);
+  const dispatch = useDispatch();
+  const editHandler = useCallback(
+    dataToEdit => {
```

Comment:
NOTE: this is where the setup happens. This builds the return URL and then opens the log in dialog

Reply:
NOTE: The source that we dump can be changed to local/session/cookie if this is too wild.